workspaces/building-a-workspace/README.md

```diff
@@ -26,63 +26,88 @@ Deleting a component may impact other components. If this is the case, you'll be
 
 ## Understanding how components work
 
-### The Text Component
+### Section component
 
-Write anything you wish to display, then customize how it's displayed using the style options.
+The section component lets you regroup multiple components into a single area.
 
-### The Action Component
+### Tabs Component
 
-The **Action** component can be used to trigger `global,` `single`, and `bulk` Smart Actions as well as CRUD operations.&#x20;
+Like the [section component](./#section-component), the tabs components lets you regroup multiple components into a single area, only the active tab components are displayed.
+The tabs component can hold between 1 and 10 tabs.
```

Comment:
Your new line is not kept in the documentation. It would be nice to have it working 🙏